ibis/backends/flink/registry.py

```diff
@@ -67,6 +67,11 @@ def _literal(translator: ExprTranslator, op: ops.Literal) -> str:
     return translate_literal(op)
 
 
+def _zeroifnull(translator: ExprTranslator, op: ops.Literal) -> str:
```

Comment:
FYI we are [considering soft-deprecating](https://github.com/ibis-project/ibis/issues/7147#issuecomment-1724249012) the `zeroifnull()` API in favor of `fillna(0)`. It's still okay to implement translation for it though!